Framework/Types/src/Core/DateAndTime.cpp

```diff
@@ -10,6 +10,8 @@ const uint32_t DateAndTime::EPOCH_DIFF = 631152000;
 /// The epoch for GPS times.
 const boost::posix_time::ptime
     DateAndTime::GPS_EPOCH(boost::gregorian::date(1990, 1, 1));
```

Comment:
I know this wan't you, but I think the GPS_EPOCH above is wrong. GPS epoch 5th Jan 1980. Just a note, shouldn't really affect anything, but might cause confusion for future maintainers.